pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -623,6 +630,22 @@ func (as *availabilitySet) ensureHostInPool(serviceName string, nodeName types.N
 		}
 	}
 	if !foundPool {
+		if as.useStandardLoadBalancer() && len(newBackendPools) > 0 {
+			// Although standard load balancer supports backends from multiple availability
+			// sets, the same interface couldn't reference more than one load balancer of
```

Comment:
better to use full name  'interface' -> 'network interface', i got a bit confused when looking at the comment.

Reply:
so are you referring go interface? that make sense